### ISSUE TITLE
v1.2.19 hangs

### ISSUE THREAD

**Issue by sthulin** [007revad/Synology_HDD_db#25]
I replaced my flash drives with new ones and went to run the latest version of the script and now it just hangs.

```
sudo ./syno_hdd_db.sh 
Synology_HDD_db v1.2.19
DS1817+-j DSM 6.2.4 
```

**Comment by sthulin** [comment]
update, it sat for several minutes before finally completing

**Comment by 007revad** [comment]
It looks it was waiting for github to reply if there was a newer version. I'll a timeout so if there's reply within 10 seconds it continues the rest of the script.

**Comment by 007revad** [comment]
Fixed in v1.2.20

- When checking if there's a newer version it now has a connection timeout of 5 seconds and an overall timeout of 10 seconds.
- When downloading a newer version it now has a connection timeout of 5 seconds and an overall timeout of 30 seconds.
